lnd.go

```diff
@@ -545,15 +547,17 @@ func lndMain() error {
 		return err
 	}
 	for _, restEndpoint := range cfg.RESTListeners {
-		listener, err := tls.Listen("tcp", restEndpoint, tlsConf)
+		lis, err := lncfg.TlsListenOnNormalizedAddress(restEndpoint,
```

Comment:
nit: current styling would prefer this line be broken as:
```golang
        lis, err := lncfg.TlsListenOnNormalizedAddress(
                 restEndpoint, tlsConf,
        )
```
also another instance further down